broadcasting/11_Summary.md

```diff
@@ -0,0 +1,7 @@
+As you can see converting an RTMP or RTSP stream to HLS might not be the easiest task, however, some understanding of the protocols and the usage of a couple of prefabricated elements in the Membrane Framework make up a rather compact solution, which is open for extension and modification.
```

Comment:
Hmm I would split that sentence into two shorter ones :D

Reply:
yeah :p